9999-structured-error-messages.md

```diff
@@ -1,77 +1,53 @@
-- Proposal Name: (fill me in with a unique ident, `my_awesome_feature`)
-- Start Date: (fill me in with today's date, YYYY-MM-DD)
 - NEP PR: [nearprotocol/neps#0000](https://github.com/nearprotocol/neps/pull/0000)
-- Issue(s): link to relevant issues in relevant repos (not required).
 
 # Summary
 [summary]: #summary
 
-One paragraph explanation of the proposal.
+Make error messages structured and include machine-readable error codes.
```

Comment:
Removing the structured part of this change might make the whole thing much simpler